netsim/outputs/_graph.py

```diff
@@ -134,6 +134,8 @@ def append_edge(graph: Box, if_a: Box, if_b: Box, g_type: str) -> None:
   e_attr = get_empty_box()
   for intf in (if_a, if_b):
     addr = intf.ipv4 or intf.ipv6
+    if type(addr) == bool:
```

Comment:
It's not wrong, but most code uses ```is True``` to test for this case https://github.com/ipspace/netlab/blob/63e83f8bf546562dfb02f7eee00d2f0bb1f0227e/netsim/augment/nodes.py#L248

Reply:
Sure, changed.